spec/amp-actions-and-events.md

```diff
@@ -311,22 +348,27 @@ For example, the following is possible in AMP.
   <tr>
     <th>Action</th>
     <th>Description</th>
+    <th>Required Trust</th>
   </tr>
   <tr>
     <td>play</td>
     <td>Plays the video.</td>
+    <td>High</td>
```

Comment:
This should be High only if video is unmuted. Muted videos should be playable with Low, could you create an issue and assign to me?